src/Keboola/Console/Command/OrganizationIntoMaintenanceMode.php

```diff
@@ -101,6 +104,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
                     $maintenanceMode
                 )
             );
+            if ($maintenanceMode === 'on' && $force) {
+                $output->writeln(
+                    'Checking the project for any running jobs',
+                );
+                $thereAreRunningJobs = $this->areThereRunningJobs(
+                    $manageClient,
+                    $project['id'],
+                    $input->getArgument(self::ARGUMENT_HOSTNAME_SUFFIX),
+                    $output
+                );
+                if ($thereAreRunningJobs) {
+                    continue;
```

Comment:
it would be good to inform that the project has been skipped due to jobs still running in that project